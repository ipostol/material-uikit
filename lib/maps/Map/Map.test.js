'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _Map = require('./Map');

var _Map2 = _interopRequireDefault(_Map);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('Map component', function () {

  it('should render correctly', function () {

    var props = {};

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_Map2.default, props));

    expect(wrapper).toMatchSnapshot();

    (0, _enzyme.mount)(_react2.default.createElement(_Map2.default, props));
  });
});