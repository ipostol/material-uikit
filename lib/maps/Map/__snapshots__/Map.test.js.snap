// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map component should render correctly 1`] = `
<div
  style={
    Object {
      "height": "100%",
      "position": "relative",
      "width": "100%",
    }
  }
>
  <GoogleMap
    bootstrapURLKeys={
      Object {
        "key": "AIzaSyAAFXTWAvZDjwWtFHa7sdW_-D42WVob39Q",
        "language": "en",
        "libraries": "places",
      }
    }
    debounced={true}
    distanceToMouse={[Function]}
    googleMapLoader={[Function]}
    hoverDistance={100}
    layerTypes={Array []}
    onChange={[Function]}
    options={
      Object {
        "maxZoom": 19,
        "minZoom": 6,
        "scrollwheel": false,
      }
    }
    style={
      Object {
        "height": "100%",
        "margin": 0,
        "padding": 0,
        "position": "relative",
        "width": "100%",
      }
    }
    yesIWantToUseGoogleMapApiInternals={true}
  />
</div>
`;
